lib/core/GridBlock.js

```diff
@@ -47,20 +47,20 @@ class GridBlock extends React.Component {
     );
   }
 
-  renderBlockImage(image, imageLink) {
+  renderBlockImage(image, imageLink, imageAlt) {
     if (image) {
       if (imageLink) {
         return (
           <div className="blockImage">
             <a href={imageLink}>
-              <img src={image} />
+              <img src={image} alt={imageAlt} />
             </a>
           </div>
         );
       } else {
         return (
           <div className="blockImage">
-            <img src={image} />
+            <img src={image} alt={imageAlt} />
```

Comment:
Same here

Reply:
From https://github.com/facebook/Docusaurus/pull/529#discussion_r179907079, leave as-is.